testing/run-scripts/image_make.sh

```diff
@@ -37,15 +37,23 @@ cat <<EOF > $TMP_DIR/Dockerfile
 FROM phusion/baseimage:0.9.19
 
 RUN apt-get -qq update
-RUN apt-get -qq install wget unzip xvfb fvwm supervisor iptables x11vnc unattended-upgrades
+RUN apt-get -qq install wget unzip bzip2 supervisor iptables unattended-upgrades
```

Comment:
Yeah it's a bit weird I don't really understand why - my best guess is it's actually the latest version of baseimage.